docs/06_how-to-guides/30_key-value-api/kv_map/10_how-to-use-kv-map.md

```diff
@@ -3,19 +3,21 @@ content_title: How-To Use Key-Value Map
 link_text: "How-To Use Key-Value Map"
 ---
 
-## Summary
+## Overview
 
-This how-to procedure demonstrates how to define and use a `Key-Value Map` (`kv map`) in your smart contract.
+This how-to demonstrates how to define and use a `Key-Value Map` (`kv map`) in a smart contract.
 
 To accomplish this task use `eosio::kv::map` template class, specify the name for the map object instantiated, the type of the key, and the type for the values stored for each key. The types used for the key and the values can be any standard type, or a user defined type.
```

Comment:
specify the name for the map object instantiated, - this doesn't read correctly - not sure what is meant here ...

Is it - Use `eosio::kv::map` class template and declare an object of this class using the template parameters to define the name of the instantiated map object, the key type and the value type?

Reply:
+ in the procedure section:

Specify "kvmap"_n, which is an eosio::name, as the first parameter, to name for the map object.

Do you need "for" here?

Reply:
+ regarding the name - "kvmap" - is there anyway which defines where/how this name is used - is it just an internal reference to distinguish one map from another?

Reply:
+ one more thought - re c++ access definers i.e. public, private protected - is there anywhere which explicitly explains how these affect access on a smart contract -  do you think this is needed?

Reply:
yeah this messy, how about putting it this way

To accomplish this task do the following:

1. Instantiate an object of type `eosio::kv::map`.
2. Specify the name for the `eosio::kv::map` object instantiated.
3. Specify the type for the map's key.
4. Specify the type for the values stored for each key.
5. The key and the values types can be of any standard type, or a user defined type.

Reply:
> * regarding the name - "kvmap" - is there anywhere which defines where/how this name is used - is it just an internal reference to distinguish one map from another?

yes it is the name of the table/map.
I will add reference section to point to the reference documentation for the kv::map class.

Reply:
> * one more thought - re c++ access definers i.e. public, private protected - is there anywhere which explicitly explains how these affect access on a smart contract -  do you think this is needed? though obviously not in this how to

I don't think we cover this topic; I think it would be helpful to mention explicitly how the c++ access definers are affecting or not the smart contract actions accessibility. I will create a jira.

Reply:
Specify the name for the eosio::kv::map object instantiated. - This sentence, specifically "object instantiated"  is not a correct sentence. "Specify the name for the instantiated eosio::kv::map object " is a correct sentence for example.